docs/source/issues.rst

```diff
@@ -26,3 +25,37 @@ below accordingly:
 
    gcloud compute disks list # to list disks in GCP
    gcloud compute disks delete ${DISK_NAME}  # to delete relevant disks
+
+
+Compute cluster not properly deleted
+------------------------------------
+
+ElasticBLAST allocates a :ref:`compute cluster <elb_cluster_name>` in the cloud to perform BLAST searches. Under some circumstances it is not properly deleted. To check if the cluster is still active and delete it, please run the commands below:
+
+.. code-block:: bash
+
+   gcloud container clusters list  # to list GKE clusters
+   gcloud container clusters delete ${ELB_CLUSTER_NAME}  # to delete your cluster
+
+
+Files left in cloud storage
+---------------------------
+
+ElasticBLAST uses cloud storage to temporally store query sequences and internal logs and metadata so that they are easily accessible during its operation. Sometimes deleting these files after the search is not successful. To double check and delete them, please run the commands below:
+
+.. code-block:: bash
+
+   gsutil ls gs://${ELB_RESULTS_BUCKET}/query_batches  # to list query files
+   gsutil -m rm gs://${ELB_RESULTS_BUCKET}/query_batches/*  # to delete query files
+
+   gsutil ls gs://${ELB_RESULTS_BUCKET}/logs  # to list log files
+   gsutil -m rm gs://${ELB_RESULTS_BUCKET}/logs/*  # to delete log files
+
+   gsutil ls gs://${ELB_RESULTS_BUCKET}/metadata  # list metadata files
+   gsutil -m rm gs://${ELB_RESULTS_BUCKET}/logs/*  # to delete metadata files
+
+
+A synchronous search may shut down too early
+--------------------------------------------
+
+When doing a synchronous search (submitted with the ``--sync`` option), ElasticBLAST is continuously probing for search status to know when the search is done. When the status check times out ElasticBLAST interprets it as search failure and shuts down the cluster.
```

Comment:
> When the status check times out

What timeout is referred to here? Is it this one?
https://elbdoc--10.org.readthedocs.build/en/10/configuration.html#blast-timeout

Reply:
No, this is `kubectl` timing out when talking to the cluster.